docs/admins/assignskill.rst

```diff
@@ -0,0 +1,158 @@
+
+To enable learners to review the skills associated with a topic, you will need to create subtopics and then assign skills to the subtopics. When learners begin a practice session, they will choose the subtopic they want to practice and then review the skills within that subtopic. 
+
+1. To create a subtopic, click on the topic and then the **Subtopics editor** |subtopiceditor| button in the top-right corner.
+
+.. |subtopiceditor| image:: /images/subtopics_editor.png
+                    :scale: 30 %
+
+2. To create a new subtopic, click on **+ Add Subtopic**, enter a title and click |createsubtopic|.
+
+.. |createsubtopic| image:: /images/create_subtopic.png
+                    :scale: 40 %
+
+3. To assign skills to a subtopic, drag the skill from the list of **Uncategorized Skills** on the right side and drop it under the relevant subtopic. To remove a skill from the subtopic, drag the skill back to the Uncategorized Skills list.
+
+.. figure:: /images/subtopics_page.png
+   :alt: Creating a subtopic
+   :scale: 30 %
+
+   *Fig. 5* 
+
+4. To unassign a skill from a topic, click the |trashcan| button next to the skill in the **Uncategorized Skills** list. This will not delete the skill, but move it back to the **Unused Skills** tab in the Topics and Skills dashboard.
+
+.. |trashcan| image:: /images/trashcan.png
+              :alt: Delete skill button
+              :scale: 40 %
+
+5. To delete a subtopic, click the red **X** next to the Subtopic Title. This moves any associated skills back to the **Uncategorized Skills** list.
+
+6. To create a high-level description for the subtopic that learners will see in the practice session, click the |pencil| button next to the Subtopic Title. Enter the description in the text box under **Subtopic Page**. This will help learners decide which subtopics to practice.
+
+.. |pencil| image:: /images/pencil.png
+            :scale: 40 %
+
+.. important::
+
+   Ensure that each subtopic contains a unique list of skills, i.e., skills are not repeated in different subtopics. 
+
+7. When you have finished creating subtopics and assigning skills to them, click |savechanges|.
+
+.. |savechanges| image:: /images/save_changes.png
+                 :alt: Save Changes button
+                 :scale: 40 %
+
+.. _createquestion:
+
+Creating a question for a skill
+--------------------------------
+
+Questions can be created either through the Topic Editor or submitted by contributors. 
+
+1. To create a question from the Topic Editor, click on the topic, and then the **Questions Editor** |questioneditor| button in the top-right corner.
+
+.. |questioneditor| image:: /images/question_editor.png
+                    :scale: 30 %
+
+2. Click the **Create Question** button.
+
+3. Select the skill for which you want to create the question and click **Next**.
```

Comment:
Here, you can select multiple skills as well, maybe mention that as well?

Reply:
Done